bluemira/equilibria/file.py

```diff
@@ -91,18 +91,18 @@ class EQDSKInterface:
         ["bcentre", "Magnetic field at the reference radius", None, "T", None, "eqfile"],
         ["cplasma", "Plasma current", None, "A", None, "eqfile"],
         ["fpol", "Poloidal current function f = R*B on 1-D flux grid", None, "T.m", None, "eqfile"],
-        ["pressure", "Plasma pressure function on 1-D flux grid", None, "n.t/m^2", None, "eqfile"],
+        ["pressure", "Plasma pressure function on 1-D flux grid", None, "N.t/m^2", None, "eqfile"],
         ["ffprime", "FF' function on 1-D flux grid", None, "m.T^2/V.s/rad", None, "eqfile"],
-        ["pprime", "P' function on 1-D flux grid", None, "n.t/m^2/V.s/rad", None, "eqfile"],
+        ["pprime", "P' function on 1-D flux grid", None, "N.t/m^2/V.s/rad", None, "eqfile"],
```

Comment:
`<Unit('metric_ton * newton * second / meter ** 2 / radian / volt')>`

Reply:
To be checked [action matti]

Reply:
yeah not sure where the t came in. Looks to me like it should be N/m^2/V.s/rad